src/templates/helpers/templateHelpers.ts

```diff
@@ -171,4 +171,14 @@ Handlebars.registerHelper("trimInterfaceName", (typeName: string): string => {
   return trimInterfaceName(typeName);
 });
 
+Handlebars.registerHelper("trimPrimitiveName", (typeName: string): string => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5970468)